Day-02/python/Schloool/day_02.py

```diff
@@ -0,0 +1,39 @@
+from functools import reduce
```

Comment:
Nochmal die Lösung von Day-02? Ist nicht das erste Mal dass sowas doppelt in der PR ist und der Merge sollte trotzdem funktionieren, aber vielleicht kriegst du es ja irgendwie gefixt